setup/docker/cloud/README.txt

```diff
@@ -1,17 +1,17 @@
 ========================================================================
 PhotoPrism for Cloud Servers
-Based on Ubuntu 22.04 LTS (Jammy Jellyfish)
+Based on Ubuntu 25.04 LTS (Plucky Puffin)
 ========================================================================
 
 SOFTWARE INCLUDED
 ------------------------------------------------------------------------
 
 PhotoPrism latest, AGPL 3
-Docker CE 23.0, Apache 2
-Traefik 2.9, MIT
-MariaDB 10.10, GPL 2
-Ofelia 0.3, MIT
-Watchtower 1.5, Apache 2
+Docker CE latest, Apache 2
+Traefik 3.3, MIT
```

Comment:
Traefik v3.4 has just been released. So it would be good to upgrade it here too before merging these changes.